src/main/resources/openapi/usermanagement.yaml

```diff
@@ -124,6 +142,14 @@ paths:
         '400':
           $ref: '#/components/responses/SecurityGroupBadRequest'
 
+  /admin/roles:
```

Comment:
Perhaps this path (context) should be `security-roles` in keep with `security-groups`?
If you agree, then I think the tags (line 149) should also be updated to be similar

Reply:
Yep fair point, this will also align us with naming in the DB schema as an added bonus. Updated.